BatchedBlas: fix typos and contribute to the text (#376)
ORNL/cpp-proposals-pub: batched_blas/batched_blas.md

The P2P interface would require extra packing and unpacking of pointers,
and therefore extra overhead.  In practice, users often want to represent
a batch as a "pre-packed" array with a particular layout,

> so they do not waste time and code with the P2P interface.

Reviewer: ```suggestion
so the P2P interface would make them waste time and code.
```
